package.json

```diff
@@ -20,6 +20,14 @@
     "build": "rollup -c",
     "test": "tape -r esm test/*.js | tap-spec"
   },
+  "exports": {
+    ".": {
+      "types": "./index.d.ts",
+      "module-sync": "./lib/index.mjs",
+      "require": "./lib/index.js",
+      "default": "./lib/index.mjs"
+    }
```

Comment:
I think this should be

```suggestion
    ".": {
      "types": "./index.d.ts",
      "module-sync": "./lib/index.mjs",
      "import": "./lib/index.mjs",
      "require": "./lib/index.js"
    },
    "./package.json": "./package.json"
```

right? the default fallthrough is to remain CJS, not ESM. Anything that `require()`s and understands exports will hit `require`. If it supports `require(esm)` then it stops at module-sync. Anything that can `import` will be directed to ESM

Also adding an `exports` map is breaking on its own.